pkgs/unified_analytics/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   clock: ^1.1.1
   file: '>=6.1.4 <8.0.0'
   http: '>=0.13.5 <2.0.0'
-  intl: ^0.18.0
+  intl: ^0.19.0
```

Comment:
As a drive by comment, it may be enough to widen the range here (`'>=0.18.0 <0.20.0'`).